src/plugins/utils/LambdaEventUtils.ts

```diff
@@ -49,6 +49,15 @@ class LambdaEventUtils {
         } else if (originalEvent['detail-type'] && originalEvent.detail &&  originalEvent.version
             && Array.isArray(originalEvent.resources)) {
             return LambdaEventType.EventBridge;
+        } else if (originalEvent.Action && originalEvent.body) {
+            try {
+                const { headers } = JSON.parse(originalEvent.body);
+                if (headers[ZeitTags.DEPLOYMENT_URL_HEADER]) {
```

Comment:
It might be better to check whether or not `headers` is exist in the body. 
```js
if (headers && headers[ZeitTags.DEPLOYMENT_URL_HEADER]) {
```
It will prevent unnecessary exceptions even though it is catched below. Because throwing exception is not free operation as it requires to collect stacktrace.

Reply:
👍